splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -61,6 +61,7 @@ else if(v1==v2)
         scripts=new TreeMap<>(ddComparator);
         scripts.put(new Splice_DD_Version(sdd,1,0,0),new UpgradeScriptForFuji(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,1,1,1),new LassenUpgradeScript(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,1,1,2),new SourceCodeUpgradeScript(sdd,tc));
     }
```

Comment:
Need more work to upgrade properly. Please file a jira and assign to us. We will need to investigate upgrade issues.